webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/filters/CORSFilter.java

```diff
@@ -56,22 +56,22 @@ public void doFilter(
       ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
       throws IOException, ServletException {
     HttpServletRequest request = (HttpServletRequest) servletRequest;
-    logger.debug("CORSFilter HTTP Request: ",request.getMethod());
+    logger.debug("CORSFilter HTTP Request: ", request.getMethod());
```

Comment:
With slf4j, we should use `logger.debug("CORSFilter HTTP Request: {}", request.getMethod());`. Note the `{}`.